gradle/publish.gradle

```diff
@@ -66,10 +68,35 @@ task prepareGhCredentials {
 prepareGhPages.dependsOn groovydoc
 publishGhPages.dependsOn prepareGhCredentials
 
+task prepareGradlePluginsRepoRelease {
+    description = "Prepare the release to the Gradle Plugins Repository"
+    group = 'release'
+    doLast {
+        def secretsProperties = rootProject.file('secrets.properties')
+        if (secretsProperties.exists()) {
+            validateSecrets(secretsProperties)
```

Comment:
validateSecrets already checks essentially that the file exists. We can simplify by removing the if statement.